applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py

```diff
@@ -29,7 +29,17 @@ def CreateMapper(origin_model_part, destination_model_part, mapper_settings):
         "integration_method"         : "gauss_integration",
         "number_of_gauss_points"     : 5,
         "in_plane_morphing"                   : false,
-        "in_plane_morphing_settings"          : {}
+        "in_plane_morphing_settings"          : {},
```

Comment:
```suggestion
        "in_plane_morphing_settings" : {},
```

Reply:
Not sure from this quick review whether this is the correct number of whitespaces, but it just looks better, when it's properly aligned ;)